content-src/components/Rating/Rating.scss

```diff
@@ -0,0 +1,17 @@
+.rating-container {
+  padding-top: 10%;
+  transform: scaleX(-1);
```

Comment:
so this is a trick for the hovering css to work. it needs to be flipped so that when you hover over the svg the previous siblings highlight instead of the succeeding siblings. this was the only way i could get it to work, is there another css trick that's better?